src/main/kotlin/com/grepp/nbe1_3_team04/reservation/service/response/ReservationInfoDetailsResponse.kt

```diff
@@ -0,0 +1,35 @@
+package com.grepp.nbe1_3_team04.reservation.service.response
+
+import com.grepp.nbe1_3_team04.reservation.domain.*
+import java.time.LocalDateTime
+
+
+data class ReservationInfoDetailsResponse(
+    val courtName: String,
+    val matchTeamName: String,
+    val matchDate: LocalDateTime,
+    val participants: List<ParticipantInfoResponse>,
+    val gender: ParticipantGender,
+    val status: ReservationStatus
+) {
+    companion object {
+        fun of(
+            reservation: Reservation,
+            participants: List<Participant>,
+            matchTeamName: String
+        ): ReservationInfoDetailsResponse {
+            val participantResponses = participants.map { participant ->
+                ParticipantInfoResponse.from(participant)
+            }
+
+            return ReservationInfoDetailsResponse(
+                reservation.court.getName(),
```

Comment:
```suggestion
                reservation.court.name,
```
court 마이그레이션하면서 getName()은 없어졌습니다

Reply:
작업하던거에 court랑 team이  import가 안되어있어서 코틀린으로 변환할 때 바꿔주지 않았나보네요ㅜㅜ

놓친 부분 잘 찾아주셔서 감사합니다!